locations/spiders/curaleaf.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+from locations.items import GeojsonPointItem
+import re
+
+
+class CuraleafSpider(scrapy.Spider):
+    name = "curaleaf"
+    allowed_domains = ["curaleaf.com"]
+    start_urls = (
+        'https://curaleaf.com/locations/',
+    )
+
+    def phone_normalize(self, phone):
```

Comment:
It doesn't seem like you're calling this anywhere. Can you remove it if you aren't using it?

Reply:
my latest commit should have addressed these. let me know